src/TeamData.elm

```diff
@@ -1,138 +1,169 @@
+inputOption allianceColor allianceNumber text =
+    Input.option ( allianceColor, allianceNumber ) (Element.text text)
 
 
 view : Model -> Element.Element Msg
 view model =
     column
         [ Background.color sky
+        , Border.color black
         , padding 50
-        , spacing 40
+        , spacing 20
+        , widthXY 5 5
+        , rounded 10
         , centerX
         , centerY
-        , rounded 20
+        , Element.scale 1.5
         ]
         [ textInput model.scouterName ScouterInput "Scouter's name"
-        , radio
+        , radioRow
             [ padding 10
             , spacing 20
-            , Font.size 60
+            , Font.size 10
             ]
             { onChange = Station
-            , selected = Just model.station
-            , label = Input.labelAbove [ Font.size 60, padding 10, spacing 20 ] (text "Which station?")
+            , selected = model.station
+            , label = Input.labelAbove [] (text "stations")
             , options =
-                [ Input.option Blue1 (text "Blue 1")
-                , Input.option Blue2 (text "Blue 2")
-                , Input.option Blue3 (text "Blue 3")
-                , Input.option Red1 (text "Red 1")
-                , Input.option Red2 (text "Red 2")
-                , Input.option Red3 (text "Red 3")
+                [ inputOption GetMatch.Blue GetMatch.One "Blue 1"
+                , inputOption GetMatch.Blue GetMatch.Two "Blue 2"
+                , inputOption GetMatch.Blue GetMatch.Three "Blue 3"
+                , inputOption GetMatch.Red GetMatch.One "Red 1"
+                , inputOption GetMatch.Red GetMatch.Two "Red 2"
+                , inputOption GetMatch.Red GetMatch.Three "Red 3"
                 ]
             }
-        , textInput (unwrapToString model.match) MatchInput "Match number"
+        , textInput model.matchNumber MatchInput "Match number"
         , Element.el
             [ Background.color orange
-            , width fill
+            , width <| minimum 350 <| fill
+            , height fill
             , center
-            , rounded 10
             , Font.color white
             , Font.glow blue 5
-            , Font.size 60
+            , Font.size 10
+            , rounded 3
+            , Font.size 15
             , Font.family
                 [ Font.external
                     { name = "Open Sans"
                     , url = "https://fonts.googleapis.com/css?family=Open+Sans:700i&display=swap"
                     }
                 ]
             ]
-            (Element.text <| team model)
+            (getTeam2 model
+                |> Result.map String.fromInt
+                |> merge
+                |> Element.text
+            )
         ]
 
 
-stationToString : Stations -> String
-stationToString station =
-    case station of
-        Blue1 ->
-            "Blue 1"
+getTeam2 : Model -> Result String Int
+getTeam2 model =
```

Comment:
Why is it `getTeam2`?
If you have another file with the `getTeam` function, you can not expose it and use it with `module.getTeam`.
Or, change the function name to be better.